<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-1LGTTLAP" failures="1" tests="1" name="Default test" time="25.189" errors="0" timestamp="2025-08-22T13:26:11 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="22.873">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepdefinitions.PlaceOrderStepDefinition.the_user_enter_username_as_and_password_as_and_click_login_button(PlaceOrderStepDefinition.java:43)
at âœ½.the user enter username as "standard_user" and password as "secret_sauce1" and click login button(file:///C:/Users/preeta/Desktop/capgemini/BDD_SauceDemo/src/test/resources/features/placeorder.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
